pkgs/development/compilers/moonplus/default.nix

```diff
@@ -0,0 +1,24 @@
+{ fetchFromGitHub, stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "MoonPlus";
```

Comment:
This should be `pname` instead. By default `name` is `${pname}-${version}`, by overriding it you end up with a derivation name that doesn't contain the version.